### DISCUSSION TITLE
Tables\Columns\TextColumn::make('something')->sticky(),

### DISCUSSION THREAD

**Discussion by Michael140020** [filamentphp/filament#12129] [Ideas]
For long sets of column rows, this should be helpful to set one column of your choice to be sticky.

**Comment by abdulsalamamtech** [comment]
This is great.

**Reply by tahinuralam** [reply]
Very useful

**Comment by ojdon** [comment]
This would be incredibly useful. Would require a bit of a refactor as the styling is on the table row rather than in a cell or table header.

**Comment by mariusjp** [comment]
This would be a great addition indeed.
